src/components/searchPanel/searchPanel.scss

```diff
@@ -0,0 +1,13 @@
+.main__search-panel {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    background-image: url(https://mijnboeken.be/images/header-background.jpg);
```

Comment:
better to save images and fonts direct in your project directory

Reply:
Will be done!